routes/feeds.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+
+const { getFeeds } = require("../contorollers/feed");
+const { getFeedsLocation } = require("../contorollers/location");
+
+router.get("/", getFeeds);
+
+router.get("/locations", getFeedsLocation);
```

Comment:
```suggestion
router.get("/", getFeeds);
router.get("/locations", getFeedsLocation);
```
개인적인 의견이지만 개행을 제거해도 괜찮지 않을까요~?